hikari/impl/rest.py

```diff
@@ -120,6 +120,8 @@ class BasicLazyCachedTCPConnectorFactory(rest_api.ConnectorFactory):
 
     def __init__(self, **kwargs: typing.Any) -> None:
         self.connector: typing.Optional[aiohttp.TCPConnector] = None
+        kwargs.setdefault("enable_cleanup_closed", True)
```

Comment:
so um, didn't this cause a windows bug earlier

Reply:
Looks like not having it is worst, lol. After all this changes I try to replicate it and I went 50 bot restarts without an issue, so idk where nor how, but its fixed